src/types/Newsroom.ts

```diff
@@ -69,6 +69,7 @@ export interface NewsroomRef {
         manual_subscription_management: string;
         privacy_settings: string;
         widget_settings: string;
+        widget_feed: string;
```

Comment:
@marko-ogg i was able to construct iframe locally, so i think using js widget should be ok